explorer/puller/db/Databases.py

```diff
@@ -0,0 +1,31 @@
+import psycopg2
+
+from .NemDatabase import NemDatabase
+
+class Databases:
+	"""Container of all databases."""
+
+	def __init__(self):
+		"""Creates a databases object."""
+
+		self._nem_db_connection = None
+		self.nem = None
+
+	def __enter__(self):
+		"""Connects to databases."""
+
+		self._nem_db_connection = psycopg2.connect(
+			database="nem_db",
+			user='user',
+			password='admin',
+			host='localhost',
+			port= '54320'
+		)
+
+		self.nem = NemDatabase(self._nem_db_connection)
```

Comment:
This seems to mimic optin puller db, but this might not be the best idea.
The reason optin db was made this way is that there are multiple databases and in multiple places, we want to access multiple of them.

Here however, I imagine, we'd rather want abstract certain operations and have similar classes for both Nem and Symbol. It really begs to use inheritance in this case.

Consider following:
```py
class DatabaseConnection:
	def __init__(self, connection_string):
		self.connection_string = connection_string

	def __enter__(self):
		print(f'opening connection {self.connection_string}')
		return self

	def __exit__(self, exc_type, exc_value, traceback):
		print(f'closing connection {self.connection_string}')

class NemDatabase(DatabaseConnection):
	def create_tables(self):
		print('creating tables')
```

This looks quite similar, but what differs is usage later:
```py
with NemDatabase('...') as db:
	db.create_tables()
```

moreover, you can later hide the guts behind some kind of Facades (note this is pretty dumb example):
```py
class NemPullerFacade:
  Db = NemDatabase
  Client = NemClient

class SymbolPullerFacade:
  Db = SymbolDatabase
  Client = SymbolClient
  ```
  
  with this in place, you'll later be able to have chain-agnostic usage:
  ```py
      client = facade.Client(...)
      
      with facade.Db() as db:
        db.nem.create_tables()

        db_height = db.get_current_height()
        chain_height = await client.height()
        ...
```

:)